docs/src/css/custom.css

```
@@ -1,71 +1,220 @@
- * Techy theme for Obot docs
+ * Obot docs theme
+ *
+ * Brand palette: purple #380067, blue #4f7ef3, teal #2ddcec, red #ff4044,
+ * yellow #fdcc11, green #06eaa7, orange #ff7240.
+ *
+ * Blue carries the theme. The blue ramp is the same one the product UI uses
+ * (ui/user/src/app.css), and the dark neutrals and accent blue are sampled from
+ * the marketing site at https://obot.ai, so all three surfaces agree.
  */
 
 @import url('https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&family=JetBrains+Mono:wght@400;500&display=swap');
```

Stylelint (17.14.0)

[error] Expected "url('https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&family=JetBrains+Mono:wght@400;500&display=swap')" to be "'https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600;700&family=JetBrains+Mono:wght@400;500&display=swap'" (import-notation)

(import-notation)